### DISCUSSION TITLE
Share your mpv.net configurations

### DISCUSSION THREAD

**Discussion by bitsper2nd** [mpvnet-player/mpv.net#260] [General]
This a thread for sharing configurations and scripts for mpv.net. I will start. apart from changing several keybindings, I am using the following scripts:

- [ontop-playback](https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/ontop-playback.lua)
- [modern-osc](https://github.com/cyl0/mpv-osc-morden-x)
- [sponsorblock](https://codeberg.org/jouni/mpv_sponsorblock_minimal)
- [control buttons near the volume indicator thingy](https://github.com/x0wllaar/MPV-SMTC)

**Comment by asianmusicguy** [comment]
@bitsper2nd  how did you get modern osc working I'm having the following issues  epically making the osc disappear on pause i really want these to play nice together

**Reply by bitsper2nd** [reply]
@asianmusicguy download the latest version of the custom osc. I added osc=no to my mpv.conf file.

**Comment by asianmusicguy** [comment]
@bitsper2nd  thank you but I also expected the following issues any thoughts? https://github.com/cyl0/MordenX/issues/6

**Comment by asianmusicguy** [comment]
should I  just download whats in your github @bitsper2nd  or would that be extrema for my case

**Reply by bitsper2nd** [reply]
No, don't do that. It wont help. I'm currently away from my computer.

**Comment by asianmusicguy** [comment]
@bitsper2nd  if you would be so kind we can maybe work together on this as im not sure where to go or where to look but if we solve this togther both things will be better for it ill wait for you

**Reply by bitsper2nd** [reply]
I'm back. The setting you are looking is for is `showonpause = true,` on the **modernx.lua** [file](https://github.com/cyl0/MordenX/blob/e58d72fb452771cf0073c473e384db5a6b64a3ee/mordenx.lua#L49). Change the `true` to `false`. The option is on the **line 49**.

![image](https://user-images.githubusercontent.com/22452474/154823994-5c8da193-4123-49a4-a595-6c9a0288b7da.png)

**Comment by asianmusicguy** [comment]
@bitsper2nd  Than you that worked however 2 questions 

1 context menu toggles appear broken  since changing osc how do i restore functionality what might be blocking it?(practically "toggle OSD" as one example)
 
2.was the method for managing the logo bug a good clan solution?

Thanks again for helping your like one of 2 people lol

**Reply by bitsper2nd** [reply]
1) Depends on your input.conf file. Check you have it in your mpv.net folder. mpv.net uses that for context menu.
2) Comment the `show logo` in **line 1994** from the **mordenx.lua** [file](https://github.com/cyl0/MordenX/blob/e58d72fb452771cf0073c473e384db5a6b64a3ee/mordenx.lua#L1994). This in plain English means; add `--` before the first letter in the line.
![image](https://user-images.githubusercontent.com/22452474/154827667-bdcbf3cb-7750-4063-b73e-a0483badbe85.png)
